### PR TITLE
Add MissingPerson

### PR DESCRIPTION
**Pull request by JonasChair**
Created class MissingPerson.

### CODE REVIEW

File: SmartVision/WindowsForms/MissingPerson.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
```
Comment (DeividasBrazenas) [inline SmartVision/WindowsForms/MissingPerson.cs]: I believe that some using directives here are unnecessary. Not a major thing, but let's keep our code clean.

File: SmartVision/WindowsForms/MissingPerson.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+namespace WindowsForms
+{
+    class MissingPerson
+    {
+        string name;
+        string surname;
+        string description;
+        DateTime birthday;
+        Image photo;
```
Comment (DeividasBrazenas) [inline SmartVision/WindowsForms/MissingPerson.cs]: Maybe we also should have a contact email or smth. to ping, when a missing person is found? We can implement it later, but there will be less files to change if we add it now

Comment (JonasChair) [reply SmartVision/WindowsForms/MissingPerson.cs]: Well the missing person is probably not the person we gonna ping to contact, it should be the user who adds the person, so kinda different classes. But yes a good idea for the future.

File: SmartVision/WindowsForms/MissingPerson.cs (hunk 1 above)
Comment (JonasChair) [reply SmartVision/WindowsForms/MissingPerson.cs]: Yes i kinda noticed that after commit. 👍

File: SmartVision/WindowsForms/MissingPerson.cs (hunk 2 above)
Comment (DeividasBrazenas) [reply SmartVision/WindowsForms/MissingPerson.cs]: Ok. So in the future we'll need to make a reference to another class, smth like ContactPerson,

**Pull request merged**